articles/virtual-machine-scale-sets/virtual-machine-scale-sets-instance-protection.md

```diff
@@ -13,7 +13,7 @@ ms.custom: avverma
 # Instance Protection for Azure Virtual Machine Scale Set instances
 
 > [!NOTE]
```

Comment:
SInce the doc now covers Flex and Uniform, maybe just remove this note?

Reply:
removing the note